src/db/mod.rs

```diff
@@ -0,0 +1,10 @@
+pub mod options;
+pub mod traits;
+pub mod types;
+pub mod utils;
+
+mod db_base;
+mod reader_base;
+
+pub use db_base::DB;
+pub use reader_base::ReaderBase;
```

Comment:
is the main responsibility of this package is to have common logics for db?

i was thinking if there is better name since the whole repo is `db`

Reply:
I think the naming is already change in @matjazv PR.

Reply:
Yes, in my PR I've already made some renamings.
If something else needs to be renamed you can leave a comment in my PR.

Reply:
ah yes, i just saw that! all good =)